internal/suites/example/compose/traefik2/docker-compose.pathprefix.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
```

Comment:
Do you really need to create a new file? Cannot we simply template the original file with ${PathPrefix}. In the default case the env var would be empty.

Reply:
Yep you should be right here, good catch!